liberty-maven-plugin/src/it/kernel-install-feature-test/prepare-multiple-features-it/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.openliberty.tools.it</groupId>
+        <artifactId>kernel-install-feature-tests</artifactId>
```

Comment:
maybe you should change the artifactId here so it is not the same as another test?

Reply:
nm...that is referencing the parent pom. I did not notice that.